x-pack/plugins/maps/public/shared/layers/sources/client_file_source/create_client_file_source_editor.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import React from 'react';
+import { JsonUploadAndParse } from '../../../../../../file_upload/public';
+
+export function ClientFileCreateSourceEditor({
+  previewGeojsonFile,
+  boolIndexData = false,
```

Comment:
Can you rename `boolIndexData`? What does that signal?

Reply:
Renamed to `isIndexingTriggered`